common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Language.scala

```diff
@@ -35,12 +35,15 @@ object Language {
       )
       .getLines
       .map(_.split(",", 2).toList)
-      .map { case List(code, label) => (code, label) }
+      .map { case List(code, label) => (code, label.replace("\"", "")) }
       .toList
 
   lazy private val languageCodeMap =
     languageCodes.toMap
 
   lazy private val languageLabelMap =
-    languageCodes.map { case (code, label) => (label, code) }.toMap
+    languageCodes
+      .flatMap { case (code, label) =>
+          label.split(";").toList.map(label => (label.trim, code))
```

Comment:
Might be worth an example here of the line in the CSV you're trying to parse? The Dutch/Flemish example in the pull request comment was good!